fb.cabal

```diff
@@ -50,45 +51,46 @@ library
   exposed-modules:
     Facebook
   other-modules:
-    Facebook.Types
-    Facebook.Monad
-    Facebook.Base
     Facebook.Auth
-    Facebook.Pager
+    Facebook.Base
+    Facebook.FQL
     Facebook.Graph
+    Facebook.Monad
     Facebook.Object.Action
-    Facebook.Object.FriendList
     Facebook.Object.Checkin
-    Facebook.Object.User
-    Facebook.Object.Page
+    Facebook.Object.FriendList
     Facebook.Object.Order
+    Facebook.Object.Page
+    Facebook.Object.User
+    Facebook.Pager
     Facebook.RealTime
-    Facebook.FQL
     Facebook.TestUsers
+    Facebook.Types
   build-depends:
       base                 >= 4       && < 5
-    , lifted-base          >= 0.1     && < 0.3
-    , bytestring           >= 0.9     && < 0.11
-    , text                 >= 0.11    && < 1.3
-    , transformers         >= 0.2     && < 0.6
-    , transformers-base
-    , monad-control
-    , resourcet
-    , data-default
-    , http-types
-    , http-conduit         >= 2.0     && < 2.2
-    , attoparsec           >= 0.10.4  && < 0.14
-    , unordered-containers
     , aeson                >= 0.5
```

Comment:
This should be bumped up to `aeson >= 0.7` since that is when the function `encodeToTextBuilder` was introduced (which you use in this PR)

  - https://hackage.haskell.org/package/aeson-0.6.2.1/docs/Data-Aeson-Encode.html
  - https://hackage.haskell.org/package/aeson-0.7.0.0/docs/Data-Aeson-Encode.html